node-tests/e2e/fastboot-support-test.js

```diff
@@ -110,6 +110,30 @@ describe('e2e: fastboot integration', function () {
     });
   });
 
+  describe('scenario: with reportOnly = false', function () {
+    before(async function () {
+      await setConfig(testProject, { reportOnly: false });
+      await testProject.runEmberCommand('build');
+      await startServer();
+    });
+
+    after(async function () {
+      await stopServer();
+      await removeConfig(testProject);
+    });
+
+    it.only('sets CSP header if served via FastBoot', async function () {
```

Comment:
This seems to be a left-over from debugging.

```suggestion
    it('sets CSP header if served via FastBoot', async function () {
```